components/demo/demo-page.js

```diff
@@ -0,0 +1,47 @@
+import './demo-snippet.js';
+import './code-view.js';
+import '../colors/colors.js';
+import '../typography/typography.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+document.body.classList.add('d2l-typography');
+
+(async() => {
+	const fontsPromise = document.fonts ? document.fonts.ready : Promise.resolve();
+	await Promise.all([
+		customElements.whenDefined('d2l-demo-page'),
+		fontsPromise
+	]);
+	document.body.removeAttribute('unresolved');
+})();
+
+class DemoPage extends LitElement {
+
+	static get properties() {
+		return {
+			title: { type: String, reflect: true }
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+			}
+		`;
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		const title = document.createElement('title');
+		title.textContent = this.title;
+		document.head.insertBefore(title, document.head.firstChild);
```

Comment:
I was thinking this would turn into a `<h1>` on the page, not the `<title>` element itself. :) I guess it could do both!

Reply:
Yeah I can add that.